testing/scenario_app/compile_ios_jit.sh

```diff
@@ -61,6 +61,7 @@ echo "Creating stub App using $SYSROOT..."
 
 echo "static const int Moo = 88;" | xcrun clang -x c \
   -arch x86_64 \
+  -fembed-bitcode \
```

Comment:
I believe this shoul djust be `-fembed-bitcode-marker` right?

Reply:
Thanks, I didn't know the difference!